.github/workflows/pr-check_markdownlint.yml

```
@@ -0,0 +1,34 @@
+name: Markdownlint (PR files)
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**/*.md'
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v29.0.0
```

```suggestion
        uses: tj-actions/changed-files@v29.0.0
        with:
          files: |
            **/*.md
```
We have to ensure that we feed only `.md` files to the linter. There are other formats in the repo that could be pulled: .html, .png, _redirects.json etc. The linter does lint and throw errors for other formats…

---

Thanks, I'm not sure why that didn't get copied across

---

Oops, I think the indenting is off, I'll rebase

---

NVM, `with` is at the right level